### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RyzenDew**
Add instructions to make users lives easier

### CODE REVIEW

**Review by 0x346e3730 — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -55,7 +55,33 @@ Run `xattr -d com.apple.quarantine path-to-whisky`
...
+- Make sure the brew command is on your path:
+which brew
+If this command does not print `/usr/local/bin/brew`, you must either modify your PATH to put `/usr/local/bin` first, or fully specify the path to brew in the subsequent commands.
+- Tap the Apple Homebrew tap, which can be found at https://github.com/apple:
+
+brew tap apple/apple http://github.com/apple/homebrew-apple
+- Install the game-porting-toolkit formula. This formula downloads and compiles several large software projects. How long this takes will depend on the speed of your computer.
+`brew -v install apple/apple/game-porting-toolkit`
+- If during installation you see an error such as “Error: game-porting-toolkit: unknown or unsupported macOS version: :dunno”, your version of Homebrew doesn’t have macOS Sonoma support. Update to the latest version of Homebrew and try again.
+`brew update brew -v install apple/apple/game-porting-toolkit`
```
[the remark this answers is not in the document]
Comment (RyzenDew) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -55,7 +55,33 @@ Run `xattr -d com.apple.quarantine path-to-whisky`
...
+
 - Install freetype `/usr/local/bin/brew install freetype`
-- Restart Whisky and remake the bottle
+
+- The Game Porting Toolkit runs under Rosetta 2. Ensure that Rosetta 2 is installed.
+`softwareupdate --install-rosetta`
+
+- Make sure the brew command is on your path:
+which brew
+If this command does not print `/usr/local/bin/brew`, you must either modify your PATH to put `/usr/local/bin` first, or fully specify the path to brew in the subsequent commands.
```
Comment (Marsgames) [inline README.md:74]:
```suggestion
If this command does not print `/usr/local/bin/brew`, you must either modify your PATH to put `/usr/local/bin` first, or fully specify the path to brew in the subsequent commands.
`export PATH=/usr/local/bin:${PATH}`
```

**Review by Marsgames — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -55,7 +55,33 @@ Run `xattr -d com.apple.quarantine path-to-whisky`
...
+`/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
+
 - Install freetype `/usr/local/bin/brew install freetype`
-- Restart Whisky and remake the bottle
+
+- The Game Porting Toolkit runs under Rosetta 2. Ensure that Rosetta 2 is installed.
+`softwareupdate --install-rosetta`
+
+- Make sure the brew command is on your path:
+which brew
```
Comment (Wizjonero) [inline README.md:73]: Change to `which brew` so it stands out more and is more readable.

**Review by Wizjonero — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -55,7 +55,33 @@ Run `xattr -d com.apple.quarantine path-to-whisky`
...
+`export PATH=/usr/local/bin:${PATH}`
+
+## brew tap apple/apple http://github.com/apple/homebrew-apple
+- Install the game-porting-toolkit formula. This formula downloads and compiles several large software projects. How long this takes will depend on the speed of your computer.
+`brew -v install apple/apple/game-porting-toolkit`
+- If during installation you see an error such as “Error: game-porting-toolkit: unknown or unsupported macOS version: :dunno”, your version of Homebrew doesn’t have macOS Sonoma support. Update to the latest version of Homebrew and try again.
+`brew update && brew -v install apple/apple/game-porting-toolkit`
+
+## run this when done
+ - `ditto /Volumes/Game\ Porting\ Toolkit-1.0/lib/ brew --prefix game-porting-toolkit /lib/`
```
Comment (victor141516) [inline README.md:84]:
```suggestion
 - ``ditto /Volumes/Game\ Porting\ Toolkit-1.0/lib/ `brew --prefix game-porting-toolkit`/lib/``
```

**Review by victor141516 — CHANGES_REQUESTED** [review]

**Comment by Marsgames** [comment]
Maybe until #13 is merged, we can add a section to help people wanting to update windows version (that should avoid some issues)

We can add something like that: 
```Markdown
### Update Windows version (from 1909 to whatever you want)
Open regedit program from your wine drive
`C:/windows/regedit.exe`

Then look for `HKEY_LOCAL_MACHINE/Software/Microsoft/Windows NT/CurrentVersion` and replace `CurrentBuild` and `CurrentBuildNumbere` by `whatever version you want`
```

**Comment by Marsgames** [comment]
Also we can add a link to releases with a bit of text to fix issues like #65

File: README.md [hunk 5]
```diff
@@ -55,7 +55,33 @@ Run `xattr -d com.apple.quarantine path-to-whisky`
 
 There is an issue where the libraries for font rendering are not properly bundled yet. You will need to install freetype.
 
-- Open an x86 terminal session with `arch -x86_64 zsh`
-- Install brew `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"` (If you run into issues, let me know)
+
+### Setup your development and Homebrew environment
```
Comment (riverar) [inline README.md:59]: These are steps that need to be followed in order. So convert them from unordered bullets to ordered numerals.

File: README.md [hunk 6]
```diff
@@ -55,7 +55,33 @@ Run `xattr -d com.apple.quarantine path-to-whisky`
...
+- Ensure the Command Line Tools for Xcode 15 beta are installed. Visit https://developer.apple.com/downloads to download these tools.
+- Enter an x86_64 shell to continue the following steps in a Rosetta environment. All subsequent commands should be run within this shell.
+arch -x86_64 zsh
```
Comment (riverar) [inline README.md:62]: Should be a separate list item/command and appropriately formatted as a code block.

File: README.md [hunk 7]
```diff
@@ -55,7 +55,33 @@ Run `xattr -d com.apple.quarantine path-to-whisky`
...
+
 - Install freetype `/usr/local/bin/brew install freetype`
-- Restart Whisky and remake the bottle
+
+- The Game Porting Toolkit runs under Rosetta 2. Ensure that Rosetta 2 is installed.
+`softwareupdate --install-rosetta`
+
+- Make sure the brew command is on your path:
+ `which brew`
+If this command does not print `/usr/local/bin/brew`, you must either modify your PATH to put `/usr/local/bin` first, or fully specify the path to brew in the subsequent commands.
```
Comment (riverar) [inline README.md:74]: Can be reduced to a step that just tells the user what to do here unconditionally (e.g., Execute: `export PATH=...`).

File: README.md [hunk 8]
```diff
@@ -55,7 +55,33 @@ Run `xattr -d com.apple.quarantine path-to-whisky`
...
+- Install the game-porting-toolkit formula. This formula downloads and compiles several large software projects. How long this takes will depend on the speed of your computer.
+`brew -v install apple/apple/game-porting-toolkit`
+- If during installation you see an error such as “Error: game-porting-toolkit: unknown or unsupported macOS version: :dunno”, your version of Homebrew doesn’t have macOS Sonoma support. Update to the latest version of Homebrew and try again.
+`brew update && brew -v install apple/apple/game-porting-toolkit`
+
+## run this when done
+- ``ditto /Volumes/Game\ Porting\ Toolkit-1.0/lib/ `brew --prefix game-porting-toolkit`/lib/``
+
+## Running steam 
+ - If you are trying to run steam add this to arguments `-noreactlogin`
```
Comment (riverar) [inline README.md:87]: Tail end of this document falls apart. I think we can do better here by separating out the "why" from the procedures. And clean up the commands so they're cut/paste'able, etc.

Some inspiration:
* https://help.apple.com/pdf/applestyleguide/en_US/apple-style-guide.pdf
* https://support.apple.com/en-us/HT202860

File: README.md (hunk 7 above)
Comment (foldfree) [reply README.md:74]: wouldn't this conflict with the native brew arm64 brew installed in /opt/homebrew/bin/brew?

**Pull request closed without merging**